schema/schema.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE music_albums(
```

Comment:
Music album class has inputs like publish date and archived so kindly add these attributes in the schema